### PR TITLE
Jon/bigquery timestamp support

### PR DESCRIPTION
**Pull request by jontradesy**
Adding support for to convert a BigQuery `TIMESTAMP` value that does not have millisecond precision but has a timezone to be converted to a Datastore `timestamp` value.

Also adding a manifest property to the `pom.xml` to point to the Apache Beam job entry point

### CODE REVIEW

**Comment by yu-iskw** [comment]
@jontradesy Sorry for the delay of my reply. I missed the PR. I will review it.

File: pom.xml [hunk 1]
```diff
@@ -96,6 +96,15 @@
...
         <version>${maven-jar-plugin.version}</version>
+        <configuration>
+          <archive>
+            <manifest>
+              <addClasspath>true</addClasspath>
+              <classpathPrefix>lib/</classpathPrefix>
+              <mainClass>com.github.yuiskw.beam.BigQuery2Datastore</mainClass>
+            </manifest>
+          </archive>
+        </configuration>
```
Comment (yu-iskw) [inline pom.xml]: Can you remove the part? Because it would be nice to separate supporting another timestamp format from specifing the main class.

Comment (jontradesy) [reply pom.xml]: Yes just removed

**Comment by yu-iskw** [comment]
@jontradesy Thank you for the contribution!!!

**Comment by yu-iskw** [comment]
@jontradesy Your PR is included in 0.6.1.
https://github.com/yu-iskw/bigquery-to-datastore/releases/tag/0.6.1

**Pull request merged**